source/features/dim-bots.tsx

```diff
@@ -19,11 +19,17 @@ const botNames = [
 	'weblate',
 ] as const;
 
+// TODO: Remove when New view is default
 const commitSelectors = [
 	...botNames.map(bot => `.commit-author[href$="?author=${bot}"]`),
 	'.commit-author[href$="%5Bbot%5D"]', // Generic `[bot]` label in author name
 ];
 
+const newViewCommitSelectors = [
```

Comment:
I already fixed this, so it's just for future reference.

"New" is current, so it doesn't need "new"
The old one needs "legacy"
```suggestion

const commitSelectors = [
```